style.css

```diff
@@ -24,27 +17,36 @@ body {
     font-size: 20px;
     color: #929292;
     font-style: normal;
-    margin: 0px 70px 35px;
```

Comment:
@ChrysaLynn 
Since this `H3` already contains margins and for this wireframe, that's good enough for us to use 👍 . Therefore, erasing the margins here just gives it a more "out of the box" type of feeling.